plot_data.Rmd

```diff
@@ -0,0 +1,272 @@
+---
+title: "plots"
+author: "jabir"
+date: "2023-09-22"
+output: html_document
+---
+
+
+```{r}
+library(spotifyr)
+library(tidyverse)
+library(lubridate)
+library(corrplot)
+library(usethis)
+library(ggthemes)
+```
+
+Import data that was set up in the notebook `set_up_data.Rmd`. 
+
+```{r}
+all_songs_df <- readRDS(file.path("data", "all_songs.RDS"))
+```
+
+```{r}
+# grabbing every variable for further use
+top_five_all <- good_songs_df |>
```

Comment:
Any dataset that is being used here we'll want to be imported from what is made in the previous notebook (set_up.Rmd). 

Basically we would want someone to be able to run this notebook without having run the first notebook recently. Does this make sense?